gsoc/index.md

```diff
@@ -6,19 +6,17 @@ permalink: /gsoc/
 
 ## New to GSoC?
 
-[Background on GSoC: start here!](./background.html)
+[Background on GSoC: Start Here!](./background.html)
 
-## GSoC/Open Astronomy 2023
+## GSoC & Open Astronomy 2024
 
-OpenAstronomy is an umbrella organisation which collects project ideas
-from any of its members. OpenAstronomy has been a GSOC mentoring organisation since 2016 and
-it's applying to participate in 2023.
+OpenAstronomy is an umbrella organisation which collects project ideas from any of its members.
```

Comment:
I couldn't help myself and did one sentence per line. I can undo it.

Reply:
I don't have a preference beyond not having the full paragraph in one giga line, but one sentence per line is totally fine by me. Thanks!